chia/wallet/puzzles/p2_delegated_puzzle_or_hidden_puzzle.py

```diff
@@ -88,8 +86,10 @@ def calculate_synthetic_offset(public_key: G1Element, hidden_puzzle_hash: bytes3
 
 
 def calculate_synthetic_public_key(public_key: G1Element, hidden_puzzle_hash: bytes32) -> G1Element:
-    r = SYNTHETIC_MOD.run([bytes(public_key), hidden_puzzle_hash])
-    return G1Element.from_bytes(r.as_atom())
+    synthetic_offset: PrivateKey = PrivateKey.from_bytes(
+        calculate_synthetic_offset(public_key, hidden_puzzle_hash).to_bytes(32, "big")
```

Comment:
this is a bit confusing (at least to a layman). We create a private key and call it "offset".